packages/design/src/animation.mdx

```diff
@@ -0,0 +1,47 @@
+---
+name: Animation
+menu: Design
+route: /animation
+---
+
+# Animation
+### Patterns
+
+Animations in Jobber fall under categories based on their purpose:
+
+**Reveal** 
```

Comment:
If the other headers are h2, can these be h3?

Reply:
Yup, sounds good. Will go in and change the headers!